frontend/src/components/LoanFormComponent.vue

```diff
@@ -54,49 +54,63 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator';
 import LoansModule from '../store/modules/LoansModule';
 import { Loan, LoanForm } from '../types';
 import UsersModule from '../store/modules/UsersModule';
+import { ref, defineComponent, SetupContext} from '@vue/composition-api';
+
 
 // loan form component is used to user interface for the user to create a loan and is later 
 // authorized by the pundit dependency 
-@Component
-export default class LoanFormComponent extends Vue {
-  public form: LoanForm = {
-    lent_by_id: 0, //eslint-disable-line camelcase
-    loaned_by_id: 0, //eslint-disable-line camelcase
-    book_id: 0 //eslint-disable-line camelcase
-  };
-  public show: boolean = true;
+export default defineComponent({
+  name: 'LoanFormComponent',
+  setup(_ : object, {root, emit} : SetupContext) {
+    const form = {
+      lent_by_id: 0, //eslint-disable-line camelcase
+      loaned_by_id: 0, //eslint-disable-line camelcase
+      book_id: 0 //eslint-disable-line camelcase
+    } as LoanForm;
 
-  // Eventlistener that does not reload the page when executed, sets the lent by id to the 
-  // current user that has been logged in and then rerenders the loan form for the 
-  // user to recreate a loan 
+    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
+    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
+    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
+    // current user that has been logged in and then rerenders the loan form for the 
+    // current user that has been logged in and then rerenders the loan form for the 
+    // current user that has been logged in and then rerenders the loan form for the 
+    // user to recreate a loan 
+    // user to recreate a loan 
+    // user to recreate a loan 
```

Comment:
Might wanna remove these comments, they seem unnecessary 
```suggestion
    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
    // current user that has been logged in and then rerenders the loan form for the 
    // current user that has been logged in and then rerenders the loan form for the 
    // current user that has been logged in and then rerenders the loan form for the 
    // user to recreate a loan 
    // user to recreate a loan 
    // user to recreate a loan 
```